src/content/get-started/install/amazon-eks.mdx

```diff
@@ -8,101 +8,230 @@ pagination_next: get-started/install-okteto-cli
+  ```
+- Check the [available versions](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html) for AWS EKS service.
+- Verify the [supported K8s versions](release-notes.mdx) for the Okteto Helm chart version you plan to install.
 :::
 
-### Domain
+Follow Amazon's [cluster creation guide](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) for more details.
 
-You'll need sufficient access to a [subdomain](self-hosted/helm-configuration.mdx#subdomain) to add a wildcard DNS record, such as dev.example.com.
-By default, all endpoints created by Okteto for your development environments will be exposed on the wildcard subdomain you choose.
+### Create EBS CSI Addon IAM Role
 
-This guide assumes your domain is registered in [Amazon Route53](https://aws.amazon.com/route53/) service.
-You can use any DNS service you prefer, but this guide focuses specifically on Amazon Route53.
+Okteto requires the EBS CSI Addon to be installed in your cluster. To do that, you need to create its IAM Role first:
 
-### Installing kubectl
+```bash
+eksctl create iamserviceaccount \
+  --region="${AWS_REGION}" \
+  --name="ebs-csi-controller-sa" \
+  --namespace="kube-system" \
+  --cluster="${CLUSTER_NAME}" \
+  --role-name="${CLUSTER_NAME}-ebs-csi-driver-role" \
+  --role-only \
+  --attach-policy-arn="arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy" \
+  --approve
+```
 
-Follow Amazon's documentation for [installing kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html). Once installed, configure `kubectl` to [talk to your new cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
+Refer to the [AWS official documentation](https://docs.aws.amazon.com/eks/latest/userguide/csi-iam-role.html) for more details.
 
-### Install the Amazon EBS CSI storage driver
+### Deploy EBS CSI Addon
 
-Install the Amazon EBS CSI storage driver as an EKS add-on so that your cluster knows how to provision storage. Follow the [official documentation here.](https://docs.aws.amazon.com/eks/latest/userguide/managing-ebs-csi.html#adding-ebs-csi-eks-add-on)
+Once you have the EBS CSI IAM Role created, deploy the EBS CSI Addon with the following command:
 
-Verify that your storage driver has been installed and is running correctly:
+```bash
+eksctl create addon \
+  --region="${AWS_REGION}" \
+  --name="aws-ebs-csi-driver" \
+  --cluster="${CLUSTER_NAME}" \
+  --service-account-role-arn="arn:aws:iam::${AWS_ACCOUNT_ID}:role/${CLUSTER_NAME}-ebs-csi-driver-role" \
+  --force
 ```
-kubectl get pods -n kube-system | grep ebs-csi
+
+Refer to the [AWS official documentation](https://docs.aws.amazon.com/eks/latest/userguide/managing-ebs-csi.html#adding-ebs-csi-eks-add-on) for more details.
+
+### Create Load Balancer Controller IAM Role
+
+Okteto requires the Load Balancer Controller to be installed in your cluster. To do that, download the IAM Policy manifest:
```

Comment:
Should we explain why we require the Load Balancer Controller? If I have to request to someone permission to do it I would like to understand why is needed. Maybe I don't need to create it or there is other thing I can do. If I don't have that information, I'm forced to talk with Okteto to ask that